akka-docs/src/test/java/jdocs/future/FutureDocTest.java

```diff
@@ -293,7 +291,7 @@ public String call() {
             },
             system.dispatcher());
 
-    f.onComplete(new PrintResult<Try<String>>(), system.dispatcher());
+    f.onSuccess(new PrintResult<String>(), system.dispatcher());
```

Comment:
`onSuccess` was deprecated in favor of `foreach` in Scala 2.12 and removed in 2.13, so this will not work with Scala 2.13 - let's move to `foreach`?

Reply:
done，@raboof.